packages/wdio-globals/src/index.ts

```diff
@@ -0,0 +1,82 @@
+type SupportedGlobals = 'browser' | 'driver' | 'multiremotebrowser' | '$' | '$$'
+
+const globals: Map<SupportedGlobals, any> = new Map()
+const GLOBALS_ERROR_MESSAGE = 'Don\'t import @wdio/globals outside of the WDIO testrunner context'
+
+type BrowserSync = import('webdriverio').Browser<'async'>
+type ElementSync = import('webdriverio').Element<'async'>
+type MultiRemoteBrowserAsync = import('webdriverio').MultiRemoteBrowser<'async'>
+type ElementArrayImport = import('webdriverio').ElementArray
+
+declare namespace WebdriverIOAsync {
+    interface Browser {}
+    interface Element {}
+    interface MultiRemoteBrowser {}
+}
+
+declare namespace WebdriverIO {
+    interface Browser extends BrowserSync, WebdriverIOAsync.Browser { }
+    interface Element extends ElementSync, WebdriverIOAsync.Element { }
+    // @ts-expect-error
```

Comment:
why is this expect-error needed here?

Reply:
It is because I wasn't able to figure out how to properly type the Mutliremote instance to allow:
```ts
browser.click() // click for both instances at the same time
```
and
```ts
browser.browserName.click() // click for single browser instance
```